src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -713,6 +772,16 @@ bool MPL_gpu_event_is_complete(MPL_gpu_event_t * var)
     return (*var) <= 0;
 }
 
+ze_result_t ZE_APICALL zeMemFree(ze_context_handle_t hContext, void *dptr)
```

Comment:
Ref https://github.com/pmodels/mpich/pull/6461, we need make this function visible.

Reply:
I'll wait until #6461 is merged before making changes

Reply:
Added